recipes/ngs-bits/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+#build
+make build_3rdparty
```

Comment:
I don't think this should be necessary. As far as I can tell, it only builds htslib. And that may well be the cause of your macos build issues. Use the htslib package please if at all possible. Or else explain with comment why that didn't work. Thanks!